importer/main.py

```diff
@@ -1101,6 +1102,58 @@ def export_resources_to_csv(resource_type, parameter, value, limit):
         logging.error(f"Failed to retrieve resource. Status code: {response[1]} response: {response[0]}")
 
 
+def encode_image(image_file):
+    with open(image_file, 'rb') as image:
+        image_b64_data = base64.b64encode(image.read())
+    return image_b64_data
+
+
+# This function takes in the source url of an image, downloads it, encodes it,
+# and saves it as a Binary resource. It returns the id of the Binary resource if
+# successful and 0 if failed
+def save_image(image_source_url):
+    headers = {"Authorization": "Bearer " + config.product_access_token}
```

Comment:
The assumption here is that these images are coming from a different source that requires different authorization from the fhir server